espnet2/enh/loss/criterions/time_domain.py

```diff
@@ -207,3 +209,65 @@ def forward(
         )
 
         return si_snr
+
+
+class TimeDomainMSE(TimeDomainLoss):
+    def __init__(self):
+        super().__init__()
+
+    @property
+    def name(self) -> str:
+        return "time_MSE_loss"
```

Comment:
Not sure. Is it better to use time_domain_MSE_loss or TD_MSE_loss?

Reply:
ditto